watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/peracetic_acid_disinfection/peracetic_acid_disinfection.py

```diff
@@ -0,0 +1,296 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+import os
+
+from pyomo.environ import (
+    ConcreteModel,
+    units as pyunits,
+    assert_optimal_termination,
+    Expression,
+    value,
+    TransformationFactory,
+)
+
+from pyomo.network import Arc, SequentialDecomposition
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.util import get_solver
+from idaes.generic_models.unit_models import Product
+import idaes.core.util.scaling as iscale
+from idaes.generic_models.costing import UnitModelCostingBlock
+
+from watertap.core.util.initialization import assert_degrees_of_freedom
+
+from watertap.core.wt_database import Database
+import watertap.core.zero_order_properties as prop_ZO
+from watertap.unit_models.zero_order import (
+    FeedZO,
+    PeraceticAcidDisinfectionZO,
+)
+from watertap.core.zero_order_costing import ZeroOrderCosting
+
+
+def main():
+    m = build()
+
+    set_operating_conditions(m)
+    assert_degrees_of_freedom(m, 0)
+    assert_units_consistent(m)
+
+    initialize_system(m)
+
+    results = solve(m)
+    assert_optimal_termination(results)
+    display_results(m)
+
+    add_costing(m)
+    m.fs.costing.initialize()
+
+    assert_degrees_of_freedom(m, 0)
+    assert_units_consistent(m)
+    results = solve(m)
+    assert_optimal_termination(results)
+    display_costing(m)
+
+    return m, results
+
+
+def build():
+    # flowsheet set up
+    m = ConcreteModel()
+    m.db = Database()
+
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+    m.fs.prop = prop_ZO.WaterParameterBlock(
+        default={
+            "solute_list": [
+                "peracetic_acid",
+                "total_coliforms_fecal_ecoli",
+            ]
+        }
+    )
+
+    # unit models
+    # feed
+    m.fs.feed = FeedZO(default={"property_package": m.fs.prop})
```

Comment:
```suggestion
        m.fs.feed = FeedZO(property_package = m.fs.prop)
```
I don't think it's necessary to address all these deprecations now, but ig it can't hurt. This default deprecation is present in a lot of places.

Reply:
I'm finding that making this suggested change results in an error when I run the flowsheet:

    ValueError: Unexpected keyword options found while constructing '_ScalarFeedZO':
	property_package

Reply:
@tarnold17 Check what version of IDAES you are using - this should work in the latest alpha release.

Reply:
Also, if you just tried to use the commit suggestion, it might have included the spaces I accidentally put in my comment.

Reply:
@andrewlee94 - It looks like my installation of WaterTAP has IDAES version 2.0.0a2 - is that recent enough that Marcus' suggestion should work for me?

@MarcusHolly - I did not use the commit suggestion button. I just applied the change locally and tried running.

Reply:
It turns out that my IDAES installation was a bit out of date. I have updated that, and addressed the deprecation warning here.